code/_helpers/text.dm

```diff
@@ -483,7 +483,7 @@ proc/TextPreview(var/string, var/len=40)
 //Generates a clickable link which will jump the camera/ghost to the target atom
 //Useful for admin procs
 /proc/jumplink(var/atom/target)
-	if (!istype(target) || QDELETED(target))
```

Comment:
this is not redundant, it checks to ensure an atom (and not a simple datum) is passed

Reply:
If you are passing it a simple datum, then the problem which needs to be addressed is not in this proc but in the proc calling it, and this proc will merely silence the issue.

If procs silence bad usage then it will never get fixed.